commit f0d5e0eb90190c5f461b0fbfc3667f3c66fec1b7
Author: Graham Ollis <plicease@cpan.org>
Date:   Wed Feb 1 08:08:08 2017 -0500

    handle extensions

diff --git a/Makefile b/Makefile
index 9754ddf..c3dc048 100644
--- a/Makefile
+++ b/Makefile
@@ -79,10 +79,10 @@ install: bzip2 bzip2recover
 	cp -f bzip2 $(PREFIX)/bin/bunzip2
 	cp -f bzip2 $(PREFIX)/bin/bzcat
 	cp -f bzip2recover $(PREFIX)/bin/bzip2recover
-	chmod a+x $(PREFIX)/bin/bzip2
-	chmod a+x $(PREFIX)/bin/bunzip2
-	chmod a+x $(PREFIX)/bin/bzcat
-	chmod a+x $(PREFIX)/bin/bzip2recover
+	chmod a+x $(PREFIX)/bin/bzip2$(EXE)
+	chmod a+x $(PREFIX)/bin/bunzip2$(EXE)
+	chmod a+x $(PREFIX)/bin/bzcat$(EXE)
+	chmod a+x $(PREFIX)/bin/bzip2recover$(EXE)
 	cp -f bzip2.1 $(PREFIX)/man/man1
 	chmod a+r $(PREFIX)/man/man1/bzip2.1
 	cp -f bzlib.h $(PREFIX)/include
